hnn_core/batch_simulate.py

```diff
@@ -252,15 +305,39 @@ def _run_single_sim(self, param_values):
             net = calcium_model()
```

Comment:
@samadpls I'm realizing that this may be difficult to maintain in the future.

Instead of a `net_name` attribute passed to the init, could we just pass in a `Network` object directly? Then it is stored as `self.net` in the class, and you can call `net.copy()` whenever you run a single simulation

Reply:
This came to mind because I was going to recommend running a smaller network using the following command
```py
net = jones_2009_model(mesh_shape(3,3))
```
but this isn't possible with the current API